### PR TITLE
adding functional tests

### PR DESCRIPTION
**Pull request by dbu**
i added minimal functional tests to see if the bundle works as expected.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,7 +22,6 @@
     "xamin/handlebars.php": "~0.10"
   },
   "require-dev": {
-    "squizlabs/php_codesniffer": "~2.0",
```
Comment (dbu) [inline composer.json:25]: not related, but this dependency is not used.

**Comment by mfeltscher** [comment]
@dbu Could you please rebase this pull request?

**Comment by dbu** [comment]
rebased and solved the conflict on phpunit.xml.dist

File: phpunit.xml.dist [hunk 2]
```diff
@@ -17,8 +21,14 @@
         <whitelist>
             <directory>.</directory>
             <exclude>
-                <directory>Resources</directory>
+                <directory>./Resources</directory>
+                <directory>./Tests</directory>
```
Comment (mfeltscher) [inline phpunit.xml.dist:25]: Just to clarify: What is this good for? Doesn't this mean, the `Tests` directory gets ignored?

Comment (dbu) [inline phpunit.xml.dist:25]: it means that you get no code coverage reports for the Tests classes. if you want those in the report, i can remove this line.

Comment (mfeltscher) [inline phpunit.xml.dist:25]: Ah, got it! No, let's leave it this way for the moment.

**Pull request merged**